cicecore/cicedynB/infrastructure/io/io_netcdf/ice_restart.F90

```diff
@@ -84,7 +84,6 @@ subroutine init_restart_read(ice_ic)
          endif
          endif ! use namelist values if use_restart_time = F
 
-         write(nu_diag,*) 'Restart read at istep=',istep0,time,time_forc
```

Comment:
Why are you deleting this line?

Reply:
I saw that---I think this might have been an accidental delete.